pages/Configuring/Environment-variables.md

```diff
@@ -0,0 +1,64 @@
+We have [previously mentioned](../Getting-Started/Quick-start/#wrapping-the-launcher-recommended) using a wrapper to launch Hyprland. Whether you
```

Comment:
no "we". Use a neutral way of writing. E.g. "As previously mentioned"

Reply:
There are multiple occasions you use "I" on the wiki. If this is a problem, I'm going to change those too.

Reply:
yeah sure.